cs/configuring.texy

```diff
@@ -94,6 +94,14 @@ application:
 
 `scanDirs` bude nyní obsahovat pouze hodnotu `foo`.
 
+/--json
+"config": {
+	"classmap-authoritative": true
```

Comment:
To nechápu, co má společného classmap-authoritative s registrací presenerů? Jsi si jistý, že víš, co to dělá?

Reply:
Jo už chápu, co je cílem. To co chceš je vlastně spíš `optimize-autoloader` než `classmap-authoritative` (tj. vynucení dumpu classmapy i pro PSR-4). `classmap-authoritative` není na běžný vývoj moc pohodlný (protože na rozdíl od RobotLoaderu musíš rebuildnout tu classmapu ručně po přidání každé nové třídy).

Chtělo by to přidat před to nějakou vysvětlující větu, že pokud jsou presentery v balíčku, co používá PSR-0 / PSR-4 autoloading, tak je potřeba zapnout generování classmapy i pro ně volbou `optimize-autoloader`.

Reply:
Ok, to je lepší formulace. A sorry, na to, že se mi díky tomu vlastně zapíná `optimize-autoloader` automaticky jsem úplně zapomněl. Hned jak bude chvíle, tak to upravím.